src/main/java/hillel/spring/doctors/model/Appointment.java

```diff
@@ -12,6 +12,8 @@
     @Id
     @GeneratedValue
     private Integer id;
+    @Version
+    private Integer version;
```

Comment:
I cannot say that it is incorrect, but this is something to think of:
Technically in this Dctor-Appointment conjunction, the Doctor class is an aggregate object. If  you want  to read or save an Appointment you have to read or save a Doctor. 
Therefore it makes sense to have versioning on a Doctor (or on an aggregate) rather than on an Appointment.